engine.go

```diff
@@ -8,25 +8,29 @@ import (
 	"gopkg.in/src-d/go-mysql-server.v0/sql/parse"
 )
 
+const mysqlVersion = "8.0.11"
+const defaultVersionPostfix = "go-mysql-server"
+
 // Engine is a SQL engine.
 type Engine struct {
 	Catalog  *sql.Catalog
 	Analyzer *analyzer.Analyzer
 }
 
 // New creates a new Engine
-func New(c *sql.Catalog, a *analyzer.Analyzer) *Engine {
+func New(c *sql.Catalog, a *analyzer.Analyzer, versionPostfix string) *Engine {
 	c.RegisterFunctions(function.Defaults)
+	c.RegisterFunction("version", sql.FunctionN(function.NewVersion(mysqlVersion+"-"+versionPostfix)))
+
 	return &Engine{c, a}
 }
 
 // NewDefault creates a new default Engine.
 func NewDefault() *Engine {
 	c := sql.NewCatalog()
-	c.RegisterFunctions(function.Defaults)
```

Comment:
why did you remove this?

Reply:
because now `NewDefault` calls `New` constructor which registers function.

Before `NewDefault` was returning `Engine` instance so copy-paste in both constructors was required.